src/Backend/SgateShopgatePlugin/Helpers/WebCheckout.php

```diff
@@ -254,4 +258,16 @@ public function getSessionId()
 
         return session_id();
     }
+
+    public function closeRequest() {
```

Comment:
```suggestion
    public function closeRequest()
    {
```
This should fix the code style test.

I like that approach much better :+1: Hope they'll accept the NOSONAR annotation.